### ISSUE TITLE
Can't get Gentle to process any files.

### ISSUE THREAD

**Issue by webtech1980** [lowerquality/gentle#89]
Hello, I installing Gentle from source but when I try to align text from http://localhost:8765 and error pops up when I click the align button. The error occurs within the terminal.

Unhandled error in Deferred:

Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/twisted/_threads/_threadworker.py", line 46, in work
    task()
  File "/usr/local/lib/python2.7/dist-packages/twisted/_threads/_team.py", line 190, in doWork
    task()
--- <exception caught here> ---
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/threadpool.py", line 246, in inContext
    result = inContext.theWork()
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/threadpool.py", line 262, in <lambda>
    inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 118, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 81, in callWithContext
    return func(*args,**kw)
  File "serve.py", line 123, in transcribe
    gen_hclg_filename = language_model.make_bigram_language_model(ks, proto_langdir, **kwargs)
  File "/home/kahless/gentle-master/gentle/language_model.py", line 122, in make_bigram_language_model
    raise e
exceptions.OSError: [Errno 2] No such file or directory

Any Idea how I can resolve this issue based on the error message?

**Comment by strob** [comment]
Did all of the steps in "install.sh" complete successfully? It looks like
the ext/ binaries didn't compile.

**Comment by webtech1980** [comment]
Yea sorry I was real tired when I posted this. I should have provided much more info. I think your statement about the ext/ binaries is right on the money. When I first executed "install.sh" it was saying that it was missing a file from kaldi. So I tried to install that separately. But I was still getting the same error. It was at that point I noticed the directory gentle-master/ext/kaldi. The gentle-master/ext/kaldi folder was empty.  So I copied my kaldi directory to that location and went through the kaldi install process again. This time when I executed the "install.sh" file it no longer got stuck at the same point. Now when I run the "install.sh" file I get the following error.

kaldi/src/fstext/fstext-utils-inl.h:482:51:   required from here
kaldi/src/fstext/determinize-star-inl.h:569:12: warning: unused variable ‘ans’ [-Wunused-variable]
In file included from standard_kaldi.cc:22:0:
rpc.h: At global scope:
rpc.h:13:3: warning: ‘RPCStatus’ defined but not used [-Wunused-variable]
 } RPCStatus;
   ^
make: *** [standard_kaldi.o] Error 1

I'm running Ubuntu 14.04. Just to mention I'm new to Linux so there's a good change I've overlooked something.  Any Ideas how I can resolve?

**Comment by webtech1980** [comment]
Ok I figured out what went wrong. When I downloaded the repository I did so from the "Clone or download" button and selected download zip. This provided me with gentle-master.zip. After extracting I was left with the a directory by the same name "gentle-master". I think that was causing the install to fail. I deleted that directory and used git to clone the repository. This created a directory by the name of just "gentle" not "gentle-master". This time when I ran the install.sh install it completed correctly and I have a functioning program.
